SideBar.py

```diff
@@ -17,6 +17,24 @@ def get_path(self, paths):
 		except IndexError:
 			return self.window.active_view().file_name()
 
+	@staticmethod
+	def make_dirs_for(filename):
+		"""
+		Attempts to create all necessary subdirectories for `filename`.
+
+		Returns True if the tree was created, False otherwise, but doesn't
+		mean we can or cannot write to it. It simply means it was not created
+		by this function.
+		"""
```

Comment:
I don't think the comments are necessary here, the code is pretty clear without it.